roles/pxe/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+- name: Reload dnsmasq      
```

Comment:
trailing whitespace

Reply:
argh, I fixed that last night and git diff wasn't showing it so I rolled back :/